src/components/Terminal/Terminal.js

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect, useRef, useState } from "react";
 import { useHistory } from "react-router-dom";
-import { useSelector } from "react-redux";
-import { getMacaroons } from "app/selectors";
+// import { useSelector } from "react-redux";
+// import { getMacaroons } from "app/selectors";
```

Comment:
Typically I wouldn't comment out code being checked into the repo, but I think in this case it's important to leave this information here for the follow-ups on fetching the authentication credentials.